### PR TITLE
Add none country hint

### LINKED ISSUE #9991: Add `none` option to default country hint

### PR DESCRIPTION
**Pull request by maririos**
Fixes: #9991 
It also adds entries for change log including changes that happened in PR #10022

### CODE REVIEW

File: sdk/textanalytics/Azure.AI.TextAnalytics/CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,10 @@
 # Release History
 
 ## 1.0.0-preview.3 (Unreleased)
-
+- Renamed type `SentimentScorePerLabel` to `SentimentConfidenceScorePerLabel`.
+- In both `DocumentSentiment` and `SentenceSentiment` property `SentimentScores` has been renamed to `ConfidenceScores`.
+- In `LinkedEntity`, property `Id` has been renamed to `DataSourceEntityId`.
+- Added `DetectedLanguageInput.None` for user convenience, instead of passing empty string to `CountryHint`.
```
Comment (samvaity) [inline sdk/textanalytics/Azure.AI.TextAnalytics/CHANGELOG.md]: Is the class name `DetectedLanguageInput` or `DetectLanguageInput` ?

Comment (maririos) [reply sdk/textanalytics/Azure.AI.TextAnalytics/CHANGELOG.md]: You are right. Thanks! will change

File: sdk/textanalytics/Azure.AI.TextAnalytics/src/DetectLanguageInput.cs [hunk 2]
```diff
@@ -11,6 +11,12 @@ namespace Azure.AI.TextAnalytics
     /// </summary>
     public class DetectLanguageInput
     {
+        /// <summary>
+        /// A wildcard that allows to set CountryHint to None
+        /// so the servide model doesn't use any default CountryHint.
+        /// </summary>
+        public static readonly string None = "";
```
Comment (jsquire) [inline sdk/textanalytics/Azure.AI.TextAnalytics/src/DetectLanguageInput.cs]: You may want to use `const` instead of `static readonly` here.

**Review by jsquire — APPROVED** [review]

**Comment by maririos** [comment]
@chidozieononiwu  any idea why CI seems stuck here?

**Comment by chidozieononiwu** [comment]
> @chidozieononiwu Dozie Ononiwu FTE any idea why CI seems stuck here?

I really don't know it didn't even trigger the text Analytics pipeline. Perhaps closing and reopening or pushing another commit will fix this.

**Pull request merged**